Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/initialization.ads

```diff
@@ -0,0 +1,14 @@
+
+with GL.Objects.Programs;
+with GL.Objects.Vertex_Arrays;
+
+with Window_Types;
+
+package Initialization is
+
+        function Initialize(Main_Window : in out Window_Types.tWindow;
```

Comment:
I don't really care much for how much spaces you indent, but I would really like to have consistency. This indentation is much more than in your other files.